tests/js/web/NavMainComponentTest.js

```diff
@@ -9,97 +9,88 @@ $.fx.off = !$.fx.off;
+            '           <li class="nav-item">' +
+            '               <a href="#one" class="nav-link" aria-haspopup="true" aria-expanded="false" aria-controls="aria-secondary-nav">1</a>' +
+            '           </li>' +
+            '           <li class="nav-item">' +
+            '               <a href="#two" class="nav-link" aria-haspopup="true" aria-expanded="false" aria-controls="aria-secondary-nav">2</a>' +
+            '           </li>' +
+            '           <li class="nav-item">' +
+            '               <a href="#three" class="nav-link" aria-haspopup="true" aria-expanded="false" aria-controls="aria-secondary-nav">3</a>' +
+            '           </li>' +
+            '       </ul>' +
+            '   </div>' +
+            '   <div class="nav-secondary" id="aria-secondary-nav">' +
+            '       <a href="#close" data-nav-action="close">x</a>' +
+            '       <form>' +
+            '           <input type="search" placeholder="search" />' +
+            '           <button>Go</button>' +
+            '       </form>' +
+            '       <div class="nav-list" data-nav="#one">' +
+            '           <ul class="nav-items">' +
+            '               <li class="nav-item">' +
+            '                   <a href="#one_one" class="nav-link">1.1</a>' +
+            '               </li>' +
+            '           </ul>' +
+            '       </div>' +
+            '       <div class="nav-list" data-nav="#two">' +
+            '           <ul class="nav-items">' +
+            '               <li class="nav-item">' +
+            '                   <a href="#two_one" class="nav-link">2.1</a>' +
+            '               </li>' +
+            '           </ul>' +
+            '       </div>' +
+            '   </div>' +
+            '   <div class="nav-tertiary" id="aria-tertiary-nav">' +
+            '       <a href="#close" data-nav-action="close">x</a>' +
+            '       <div class="nav-list">' +
+            '           <ul class="nav-items">' +
+            '               <li class="nav-item">' +
+            '                   <a href="#three_one" class="nav-link" aria-haspopup="true" aria-expanded="false" aria-controls="aria-quaternary-nav">3.1</a>' +
+            '               </li>' +
+            '           </ul>' +
+            '       </div>' +
+            '   </div>' +
+            '   <div class="nav-quaternary" id="aria-quaternary-nav">' +
+            '       <a href="#close" data-nav-action="close">x</a>' +
+            '       <div class="nav-list" data-nav="#three_one">' +
+            '           <ul class="nav-items">' +
+            '               <li class="nav-item">' +
+            '                   <a href="#four_one" class="nav-link">4.1</a>' +
+            '               </li>' +
+            '           </ul>' +
+            '       </div>' +
+            '   </div>' +
+            '</nav>' +
+            '<div class="content-main"></div>'
```

Comment:
we should soooo use ES6 string template literals here (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). It'll mean we can build these DOM structures without all the `+`s

e.g

```
$(`<div>
    <lots>
        <of>
             <dom></dom>
        </of>
    </lots>
</div>`);